src/core/mempool.rs

```diff
@@ -653,6 +654,11 @@ impl MemPoolDB {
         // apply all migrations
         MemPoolDB::apply_schema_migrations(&mut tx)?;
 
```

Comment:
Wouldn't you want to add the index creation code snippet to the `open` function as well? Same comment for other dbs (like `open_db` for StacksChainState)

Reply:
Yes; thanks for catching that.

Reply:
Fixed